iModelCore/ECDb/CHANGES.md

```diff
@@ -0,0 +1,38 @@
+# Changes
+
+This document including important changes to syntax or file format.
+
+| Tables          | Version   |
+| --------------- | --------- |
+| Profile Version | `4.0.0.2` |
+| ECSQL Version   | `1.0.0.0` |
+
+## `11/22/2022`: Add `PRAGMA` support in `ECSQL`
```

Comment:
Can you add this info to NextVersion.md please so it's documented as part of a iTwin.js release?

Reply:
This document is intended for dev use to keep track of major changes internal or external and explain what we added or deleted and why.  Some of these cannot be put into iTwin.js release another can. I will not advertise PRAGMA stuff for external users right now. We might refine and add or delete some of these pragmas. Though there is a genuine use but it's internal.